tests/playwright-test/test-parallel.spec.ts

```diff
@@ -223,3 +223,27 @@ test('parallel mode should minimize running beforeAll/afterAll hooks 2', async (
   expect(countTimes(result.output, '%%beforeAll')).toBe(2);
   expect(countTimes(result.output, '%%afterAll')).toBe(2);
 });
+
+test('parallel mode should display worker count taking project workers into account', async ({ runInlineTest }) => {
+  const result = await runInlineTest({
+    'playwright.config.ts': `
+      export default {
+        workers: 10,
+        fullyParallel: true,
+        projects: [
+          { name: 'project1', workers: 3 },
```

Comment:
```suggestion
          { name: 'project1', workers: 3 },
          { name: 'project2', workers: 3 },
```

I don't think there is an easy fix - if you do it like this, your patch shows 3 as well while in reality its 6.

Reply:
Good point. I hadn't considered projects running in parallel.